src/prefect/client/cloud.py

```diff
@@ -57,6 +57,7 @@ def __init__(
         httpx_settings["headers"].setdefault("Authorization", f"Bearer {api_key}")
 
         httpx_settings.setdefault("base_url", host)
+        httpx_settings.setdefault("follow_redirects", True)
```

Comment:
Should both the `CloudClient` and `PrefectHttpxClient` have `follow_redirects` default to `true`?